tests/build/build.go

```diff
@@ -117,6 +120,9 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 				Expect(err.Error()).To(ContainSubstring("Reference does not exist"))
 			}
 
+			// Delete created webhook from GitHub
+			cleanupWebhooks(f, helloWorldComponentGitSourceRepoName, pacControllerHost)
```

Comment:
If we're deleting webhooks in the AfterAll block of the tests, we should ideally not need a nightly job to take care of the webhooks, right?

But I guess we have nightly jobs to do cleanups for cases like when the AfterAll block wasn't executed for some reason (like an interrupt from a user, etc) which leaves behind webhooks, correct?